tests/app/helpers/test_template_helpers.py

```diff
@@ -19,42 +19,42 @@ def test_get_footer_context_nisra_theme(app):
             "rows": [
                 {
                     "itemsList": [
-                        {
-                            "text": "Help",
-                            "url": "https://census.gov.uk/ni/help/help-with-the-questions/online-questions-help/",
-                            "target": "_blank",
-                        },
-                        {
-                            "text": "Contact us",
-                            "url": "https://census.gov.uk/ni/contact-us/",
-                            "target": "_blank",
-                        },
+                        Link(
+                            text="Help",
+                            url="https://census.gov.uk/ni/help/help-with-the-questions/online-questions-help/",
+                            target="_blank",
+                        ),
+                        Link(
+                            text="Contact us",
+                            url="https://census.gov.uk/ni/contact-us/",
+                            target="_blank",
+                        ),
```

Comment:
Python objects shouldn't be being passed to the templates, this should return as it did previously.

Reply:
What's the reason for not passing Python objects, especially as what we are passing already are objects?

Reply:
We have previously had complexity caused by complex python objects being passed to and used by templates, so we have tried to not. For that one it's also that it's using the class for two different purposes - one to configure the links for a given runner mode, and then to serialise the link context. It works as it the two use the same property names, but that ties them together and isn't clear.

It also just so happens to work for that scenario as the property names are single words, in other instances the template property names are camel cased which we wouldn't have as python prroperty names.

Reply:
Done